iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/unique_ptr.inl

```diff
@@ -76,7 +76,7 @@ T* unique_ptr<T>::operator->() noexcept
 template <typename T>
 const T* unique_ptr<T>::operator->() const noexcept
 {
-    // AXIVION Next Construct AutosarC++19_03-A5.2.3 : avoid code duplication
+    // Avoid code duplication
```

Comment:
Was this intentional? All other `const_cast` have both, Axivion and clang-tidy suppresions

Reply:
`A5.2.3` states that only that `cv` type qualifiers shall not be casted away, but not the other way round.